### ISSUE TITLE
Workspace Trust should be made accessible more easily.

### ISSUE THREAD

**Issue by yilmazdurmaz** [microsoft/vscode#174823]
Type: <b>Feature Request</b>

In a recent update, I saw a shortcut was added to easily switch Trust level.

But accessing this Page requires too many steps and maybe hitting on keyboard on the way. In case of an accidental selection of trusting, finding this page back again is not easy. 

Currently there are 2 occurences this page can be opened easily, and both requires a zero/no trust level. One occasion is when a untrusted folder is opened where the page is auto-popped. and the other is to use the shield icon that sits on the left-bottom corner.

But once you select to trust the folder, game is over. you have to first remember what is the name of this setting so to make a search. then search all the menus where you find almost no traces. then maybe make an internet search. only then open the command palette and start typing Workspace Trust.

This page also supplies a list of other trusted folder, so one can easily check them and remove some or all folders to clean the list. For such an important (and useful) setting, accessing it is too much stressful.

So, in short, please make accessing to Workspace Trust page is easier. Two of the best places I can think of are the status bar (a shield just like an untrusted workspace has) and/or a menu item under the settings menu.

VS Code version: Code 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:32:34.589Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

**Comment by yilmazdurmaz** [comment]
> Unfortunately, as the description is not in English, it's hard for us to work on. If possible, please edit the original issue to be in English.

I know you corrected its mistake, but does this bot do this often or is it new?

**Comment by lszomoru** [comment]
The bot does look at the issues to detect when the description is not in English but this was a bug in our automation.
